src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -79,171 +79,139 @@ export default defineConfig({
 
-The following example automatically generates `_routes.json` while including and excluding additional routes. Note that that is only necessary if you have custom functions in the `functions` folder that are not handled by Astro.
+```js
+import cloudflare from '@astrojs/cloudflare';
+import { defineConfig } from 'astro/config';
 
-```js title="astro.config.mjs" ins={4-8}
 export default defineConfig({
-  adapter: cloudflare({
-    mode: 'directory',
-    routes: {
-      strategy: 'include',
-      include: ['/users/*'], // handled by custom function: functions/users/[id].js
-      exclude: ['/users/faq'], // handled by static page: pages/users/faq.astro
-    },
-  }),
+	adapter: cloudflare({
+		platformProxy: {
+			enabled: true,
+			configPath: 'wrangler.json',
+			experimentalJsonConfig: true,
+			persist: './.cache/wrangler/v3',
+		},
+	}),
 });
 ```
+### routes.extend
 
-### imageService
+This option allows you to add or exclude custom patterns (e.g. `/fonts/*`) to the generated [`_routes.json`](#custom-_routesjson) file that determines which routes are generated on-demand. This can be useful if you need to add route patterns which cannot be automatically generated, or exclude prerendered routes.
 
-<p>
+More information about the custom route patterns can be found in [Cloudflare's routing docs](https://developers.cloudflare.com/pages/functions/routing/#functions-invocation-routes). Any routes specified are not automatically deduplicated and will be appended to the existing routes as is.
 
-**Type:** `'passthrough' | 'cloudflare' | 'compile'`<br />
-**Default:** `'passthrough'`
+#### routes.extend.include
+
+<p>
+**Type:** `{ pattern: string }[]`<br />
+**Default:** `undefined`
 </p>
 
-Determines which image service is used by the adapter. The adapter will default to `passthrough` mode when an incompatible image service is configured. Otherwise, it will use the globally configured image service:
+Configure additional routes to be generated on demand by the Cloudflare adapter in the `routes.extend.include` array.
 
-* **`cloudflare`:** Uses the [Cloudflare Image Resizing](https://developers.cloudflare.com/images/image-resizing/) service.
-* **`passthrough`:** Uses the existing [`noop`](/en/guides/images/#configure-no-op-passthrough-service) service.
-* **`compile`:** Uses Astro's default service (sharp), but only on pre-rendered routes at build time. During SSR for pages rendered on-demand, all `astro:assets` features are disabled.
+#### routes.extend.exclude
 
-```js title="astro.config.mjs" ins={6}
-import {defineConfig} from "astro/config";
-import cloudflare from '@astrojs/cloudflare';
+<p>
+**Type:** `{ pattern: string }[]`<br />
+**Default:** `undefined`
+</p>
+
+Configure routes to be excluded from on-demand rendering in the `routes.extend.include` array. These routes will be prerendered and served statically instead, and will not invoke the SSR function.
```

Comment:
```suggestion
Configure routes to be excluded from on-demand rendering in the `routes.extend.include` array. These static files (e.g. pages, images, text files) will be prerendered and served statically instead, and will not invoke the SSR function.
```
Is that a way to make this sentence accurate? If not, can you change it so that it is?

Reply:
```suggestion
Configure routes to be excluded from on-demand rendering in the `routes.extend.exclude` array. These routes will be prerendered and served statically instead, and will not invoke the SSR function. Additionally you can use this option to serve any static asset (e.g. images, fonts, css, js, html, txt, json, etc.) files directly without routing the request through the SSR function.
```

Not really, so it is correct that **only** routes are prerendered, but other static files can also be added to the exclude array. Something like my suggestion should be technically accurate, please just check that it sounds good :)

Reply:
Perfect, done!